include/functions.h

```diff
@@ -616,7 +616,7 @@ PosRot* Actor_GetFocus(PosRot* dest, Actor* actor);
 PosRot* Actor_GetWorld(PosRot* dest, Actor* actor);
 PosRot* Actor_GetWorldPosShapeRot(PosRot* dest, Actor* actor);
 
-s32 func_800B83F8(Actor* actor, Player* player, s32 flag);
+s32 Target_NotInLeashRange(Actor* actor, Player* player, s32 skipChecks);
```

Comment:
```suggestion
s32 Target_OutsideLeashRange(Actor* actor, Player* player, s32 skipChecks);
```
Still prefer `Outside` to avoid the negative in the function name.